flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -545,7 +549,7 @@ public void notifyCheckpointAborted(long checkpointId) throws Exception {
 				StateSnapshotTransformFactory.noTransform());
 
 			newRocksStateInfo = RocksDBOperationUtils.createStateInfo(
-				newMetaInfo, db, columnFamilyOptionsFactory, ttlCompactFiltersManager);
+				newMetaInfo, db, columnFamilyOptionsFactory, ttlCompactFiltersManager, writeBufferManagerCapacity);
```

Comment:
```suggestion
				newMetaInfo, db, columnFamilyOptionsFactory, ttlCompactFiltersManager,
				optionsContainer.getWriteBufferManagerCapacity());
```
We can directly use `optionsContainer.getWriteBufferManagerCapacity()` here and don't need to pass `writeBufferManagerCapacity` through constructor parameters.